src/mmw/apps/water_balance/templates/home_page/index.html

```diff
@@ -2,11 +2,15 @@
 
 {% load staticfiles %}
 
+{% block metatitle %}
+    <title>Model My Watershed - Micro Site Storm Model</title>
+{% endblock metatitle %}
+
 {% block header %}
     <header class="water-balance">
 
         <nav id="app-header" class="navbar-watershed">
-            <div class="brand"><a href="#">Model My Watershed</a></div>
+            <div class="brand"><a href="#">Model My Watershed - Micro Site Storm Model</a></div>
```

Comment:
I know this is sort of sneaking this in on this PR, but could you change `#` to `/` so we got a proper "home" link that works?